### ISSUE TITLE
[feat] debug logs

### ISSUE THREAD

**Issue by pkarw** [callstackincubator/cali#28]
As this app can generate totally random execution plans (steps provided vs. planned etc) it would be useful to log these plans and steps to some kind of log file/core dump so it could be provided along with the issue description

I've asked the app to "build an iOS app called Example, build and run it in the emulator" - it did some steps, but now it's erroring on `getReactNativeConf` tool:

```bash
│
▲  Something went wrong
node:internal/modules/run_main:122
    triggerUncaughtException(
    ^

TypeError: import_listDevices.default is not a function
```

I'd be happy to include some stack trace what the app did - including prompts and executed steps in order to reproduce it. 

Of course, this error probably happened only due to my RN incompetence but still :)

**Comment by grabbou** [comment]
Good idea. We already print commands executed on each step, but we should save everything to a log or print it by default at the end. Will add later today!

Creating new projects is not yet supported, but hopefully I can land it later today (related to your other issues)

**Comment by borisyankov** [comment]
Proxying the traffic through [Helicone](https://www.helicone.ai/) provides great observability and insights.
(it has a Free tier, also can be self-hosted)

There is even official integration in Vercel's AI SDK:
https://docs.helicone.ai/getting-started/integration-method/vercelai

**Comment by grabbou** [comment]
That is interesting! Let's put that on the roadmap for sure. Meanwhile, I'll make sure we print proper logs in case it breaks again in the future.
